### PR TITLE
Cannot build it from my working copy, pls help

### PR DESCRIPTION
**Pull request by KPACHbIuLLIAnO4**

### CODE REVIEW

File: plugins/all/all.go [hunk 1]
```diff
@@ -19,4 +19,5 @@ import (
 	_ "github.com/influxdb/telegraf/plugins/redis"
 	_ "github.com/influxdb/telegraf/plugins/rethinkdb"
 	_ "github.com/influxdb/telegraf/plugins/system"
+	_ "github.com/KPACHbIuLLIAnO4/telegraf/plugins/apache"
```
Comment (sparrc) [inline plugins/all/all.go]: this should be 
```
github.com/influxdb/telegraf/plugins/apache
```

File: plugins/apache/apache.go [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+	"github.com/influxdb/telegraf/plugins"
+)
+
+type Apache struct {
+	Urls []string
+}
+
+var sampleConfig = `
+# An array of Apache status URI to gather stats.
+urls = ["http://localhost/server-status?auto"]`
```
Comment (sparrc) [inline plugins/apache/apache.go]: add a line-break before the end of the string

File: plugins/all/all.go (hunk 1 above)
Comment (KPACHbIuLLIAnO4) [inline plugins/all/all.go]: Thank you for advice.

But I think I've not described the problem properly.

The problem is, when I clone https://github.com/KPACHbIuLLIAnO4/telegraf.git
with apache plugin and do "make build",  I get the same executable
"telegraf" as when I do "make build" in "
https://github.com/influxdb/telegraf" working copy, with the same size, and
when I do "./telegraf -usage apache", I get the message "2015/08/29
07:10:27 Plugin apache not found"




2015-08-29 4:33 GMT+03:00 Cameron Sparr <notifications@github.com>:

**Comment by KPACHbIuLLIAnO4** [comment]
Thank you for advice.

But I think I've not described the problem properly.

The problem is, when I clone https://github.com/KPACHbIuLLIAnO4/telegraf.git with apache plugin and do "make build",  I get the same executable "telegraf" as when I do "make build" in "https://github.com/influxdb/telegraf" working copy, with the same size, and when I do "./telegraf -usage apache", I get the message "2015/08/29 07:10:27 Plugin apache not found"

**Comment by KPACHbIuLLIAnO4** [comment]
Found out the reason, thank you

**Comment by sparrc** [comment]
@KPACHbIuLLIAnO4  what was the reason?

**Comment by KPACHbIuLLIAnO4** [comment]
I forgot to rewrite some includes on my fork to make it work, because
original repo don't have the new plugin

2015-08-31 7:00 GMT+03:00 Cameron Sparr <notifications@github.com>:

**Pull request closed without merging**